### DISCUSSION TITLE
Epoptes-Client in Fedora

### DISCUSSION THREAD

**Discussion by rkwesk** [epoptes/epoptes#170] [Q&A]
Hello All

Apologies if this issue has already been raised. I tried to check but have failed to find anything.

So, the issue: on a standalone pc with fedora 35 the system was updated on 12 October 2022 and with the sphere repo installed an attempt to install epoptes-client has failed with these messages:

"Installing:
 epoptes-client       noarch       21.02-1        rpmsphere-noarch        25 k

Transaction Summary
===============================================================================
Install  1 Package

Total download size: 25 k
Installed size: 59 k
Is this ok [y/N]: y
Downloading Packages:
epoptes-client-21.02-1.noarch.rpm               34 kB/s |  25 kB     00:00    
-------------------------------------------------------------------------------
Total                                           34 kB/s |  25 kB     00:00     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                       1/1 
  Running scriptlet: epoptes-client-21.02-1.noarch                         1/1 
Failed to enable unit: Unit file epoptes-client.service does not exist.
error: %prein(epoptes-client-21.02-1.noarch) scriptlet failed, exit status 1

Error in PREIN scriptlet in rpm package epoptes-client
  Verifying        : epoptes-client-21.02-1.noarch                         1/1 

Failed:
  epoptes-client-21.02-1.noarch"

Richard

**Comment by alkisg** [comment]
Hi Richard!

Bug reports should be able to reach the one that writes or maintains the related code, otherwise they're useless.
Personally I've never used rpmsphere or even epoptes in Fedora, and I haven't yet tried to create any rpm packages.

So it would be best to try to find the author of that rpm package and ask them about this issue.
Do also invite them to participate upstream in the epoptes github repository if they wish; then people will be able to reach them via github like they do for the Debian/Ubuntu packaging issues.

I'll convert this issue into a discussion as it doesn't involve any changes in the LTSP code base, at least until someone is willing to commit upstream code for RPM packaging.

Btw a few years ago you opened a similar issue, https://github.com/epoptes/epoptes/issues/106, but noone was interested in contributing code then either! :)

**Comment by alkisg** [comment]
Hi @rkwesk,

you may test the RPMs from there:
https://github.com/epoptes/epoptes/releases/tag/v23.08
